README.md

```diff
@@ -12,6 +12,7 @@ It is written in [Go][go-site] and is a soft fork of [Hugo][hugo-site].
 [![CircleCI](https://circleci.com/gh/gothamhq/gotham.svg?style=shield)](https://circleci.com/gh/gothamhq/gotham)
 [![codecov](https://codecov.io/gh/gothamhq/gotham/branch/master/graph/badge.svg)](https://codecov.io/gh/gothamhq/gotham)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![GoDoc](https://godoc.org/github.com/gothamhq/gotham?status.svg)](https://godoc.org/github.com/gothamhq/gotham)
```

Comment:
We don't need to do it in this PR but this should link to `pkg.go.dev` and not `GoDoc.org`.